src/store/modules/auth.js

```diff
@@ -3,22 +3,22 @@ import { io } from '../../plugins/sails.js';
 export default {
 	state: {
 		authenticated: false,
-		email: null,
+		username: null,
 		mustReauthenticate: false,
 	},
 	getters: {
 		myUserName(state) {
-			return state.email ? state.email.split('@')[0] : null;
+			return state.username ? state.username : null;
```

Comment:
We don't need this getter at all anymore now that userName is a direct piece of state. We can just use state.username itself. This'll involve refactoring the references to this getter, but it'll be cleaner this way. The only reason for it before was to strip the @w/e